requirements.txt

```diff
@@ -11,5 +11,8 @@ pyright
 pytest
 scipy
 tomli
+python-dotenv
 eth-ape
-python-dotenv
\ No newline at end of file
+ape-solidity
```

Comment:
please double check that these do not create dependency issues when pip installing. Thanks!

Reply:
It seems to completely replace the `ape plugins install .` step. Once we get there, ape just says "already installed" and skips past. Makes me wonder why both approaches exist, if they do the same thing.

Reply:
Putting them in `requirements.txt` gives you more flexibility, to pin to specific commits, or specifc branches (which I had just used before they upstreamed the recent bugfix). Which the `ape-config` parser doesn't allow. It does allow specific numbered versions.

Reply:
I'll set it back to using `ape-config` for now, but keepign in mind that we may want to use requirements.txt for exceptional situations